azure-pipelines.yml

```diff
@@ -0,0 +1,205 @@
+trigger:
+  - master
+
+pr:
+  - master
+
+variables:
+- template: eng/common-variables.yml
+
+resources:
+  containers:
+  - container: LinuxContainer
+    image: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
+
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      artifacts:
+        publish:
+          artifacts: true
+          logs: true
+          manifests: true
+      enableMicrobuild: true
+      enablePublishUsingPipelines: true
+      useBuildManifest: ${{ variables['_UseBuildManifest'] }}
+      workspace:
+        clean: all
+      jobs:
+      - job: Windows_NT
```

Comment:
can you confirm Windows_NT is correct?

Reply:
Yes https://github.com/dotnet/arcade/blob/master/Documentation/AzureDevOps/AzureDevOpsOnboarding.md#external--pool-provider-netcorepublic-pool